mlflow/utils/validation.py

```diff
@@ -62,8 +60,7 @@ def _validate_metric(key, value, timestamp, step):
     it isn't.
     """
     _validate_metric_name(key)
-    if not isinstance(value, numbers.Number) or value > np.finfo(np.float64).max \
-            or value < np.finfo(np.float64).min:
+    if not isinstance(value, numbers.Number):
```

Comment:
Is there a unit test for this guy?

Reply:
you mean validate metric? I don't think so, but I am not sure if there needs to be one, cause it's pretty much reduced to isinstance call (assuming there is a unit test for the name validation).